modules/apps/headless/headless-asset-library/headless-asset-library-impl/src/main/java/com/liferay/headless/asset/library/internal/dto/v1_0/converter/AssetLibraryDTOConverter.java

```diff
@@ -84,7 +84,7 @@ public AssetLibrary toDTO(
 						dtoConverterContext.isAcceptAllLanguages(),
 						group.getDescriptionMap()));
 				setExternalReferenceCode(group::getExternalReferenceCode);
-				setId(depotEntry::getDepotEntryId);
+				setId(depotEntry::getGroupId);
```

Comment:
This was changed back to the depot ID on purpose. The issue was that were exposing and using the group ID, but build REST would use the depot ID automatically when generating the action URLs.